pkgs/development/tools/continuous-integration/jenkins-job-builder/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl, pythonPackages, buildPythonPackage, git }:
+
+buildPythonPackage rec {
+  name = "jenkins-job-builder-${version}";
+  version = "1.2.0";
+  namePrefix = "";
+
+  src = fetchurl {
+    url = "https://pypi.python.org/packages/source/j/jenkins-job-builder/jenkins-job-builder-${version}.tar.gz";
```

Comment:
Use `namePrefix` and change `jenkins-job-builder` in url to it. And after that `jenkins-job-builder-${version}` -> `${name}`.